lib/publiccloud/instance.pm

```diff
@@ -147,7 +147,7 @@ sub _prepare_ssh_cmd {
         $cmd = "\$'$cmd'";
     }
 
-    my $ssh_cmd = sprintf('ssh -t %s "%s@%s" -- %s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd);
+    my $ssh_cmd = sprintf('ssh -tvE /var/tmp/ssh_sut.log %s "%s@%s" -- %s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd);
```

Comment:
this one is not considering someone could specify `-E something` through `ssh_opts` like sles4sap is  doing in:
-  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20169/files#diff-5404750314cf685b060d425b85a04de2cdfb000225014d52ba83700abe58b2bdR142
- evolved to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20274/files#diff-5404750314cf685b060d425b85a04de2cdfb000225014d52ba83700abe58b2bdR125

Result is that in some situations the result could have `-E something` added twice to the command line 

http://openqaworker15.qa.suse.cz/tests/307727/logfile?filename=serial_terminal.txt

Reply:
That is true.